### PR TITLE
external access to distance2Point by getXyAtVal

### PR DESCRIPTION
**Pull request by ppKrauss**
need for an inverse of getValAtXY()

### CODE REVIEW

File: src/hilbert.js [hunk 1]
```diff
@@ -96,6 +96,11 @@ export default function() {
             });
         return hilbert.point2Distance(xy[0], xy[1], n);
     };
+    
+    hilbertLayout.getXyAtVal = function(dist) {
```
Comment (vasturiano) [inline src/hilbert.js:100]: Thanks for this PR @ppKrauss!

Would this method not provide the same results as:
```
.layout({ start: dist, length: 0 })
```

And if so, do you think there's still usefulness to have it?
It seems to me that `getValAtXY` is already the reverse function of the main purpose of the module, which is to extract XY coordinates from a single value, and that is provided by `.layout(...)`.

Comment (ppKrauss) [reply src/hilbert.js]: Hi, perhaps is a kind of alias, but is **less complex to use**...  The use of `layout()` was difficult to see at README before you say (here).
It is a simple public exposition of `distance2Point(length)` method. For an input length returns a XY  as result, more simple and objective tham `layout()`.

Comment (ppKrauss) [reply src/hilbert.js:100]: (oops, I deleted last comment, now I understand better what you say) 

Hi, there are many situations in the use of  space-filling curves   where we need IJ or XY from the Cell ID. Please, your code is good as generic a library, not only for one applications, but for many applications, some that you plan or know, others that you don't imagine... But I and others need: the method is there (`distance2Point`), we just need access to the method. It is not new development, it  just improves your code **as reusable library**, and to  be more expressive  and more [orthogonal set of methods](https://en.wikipedia.org/wiki/Orthogonal_instruction_set).

Comment (vasturiano) [reply src/hilbert.js:100]: Hi, it is mildly redundant to have the same functionality under two separate methods, but I hear your point about it being a more convenient interface in some circumstances. I'll expose the additional method. 
Could you give me access permission to your PR branch so I do some small cleanups and add docs?

**Pull request merged**